library/src/meta/reference.rs

```diff
@@ -111,13 +111,21 @@ impl Show for RefElem {
             bail!(self.span(), "cannot reference {}", elem.func().name());
         }
 
+        let default_supplement = elem
+            .with::<dyn LocalName>()
```

Comment:
maybe move this into the else case and remove the let binding?

Reply:
It's changed in the latest commits